app/src/main/java/com/example/dailyandroidquickstart_kotlin/MainActivity.kt

```diff
@@ -81,15 +78,17 @@ class MainActivity : AppCompatActivity() {
                 toggleCamera.isChecked = inputSettings.camera.isEnabled
                 toggleMicrophone.isChecked = inputSettings.microphone.isEnabled
             }
-
-
         })
 
         findViewById<Button>(R.id.leave)
             .setOnClickListener {
                 Log.d("BUTTONS", "User tapped the Leave button")
                 call.leave {
-                    Log.d("Call", "Call has been left")
+                    if (it.isError) {
```

Comment:
LMK if there's a better "Kotlin-y" way to do this, I just went with what made most sense to me after looking at the return object.